kie-soup-bom/pom.xml

```diff
@@ -20,16 +20,13 @@
 
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.jboss.integration-platform</groupId>
-    <artifactId>jboss-integration-platform-parent</artifactId>
-    <!-- Keep in sync with the parent version of ../pom.xml -->
-    <version>7.0.0.Final</version>
-    <relativePath/>
+    <groupId>org.kie.soup</groupId>
+    <artifactId>kie-soup-parent</artifactId>
+    <version>8.0.0-SNAPSHOT</version>
```

Comment:
We can not do this, as this brings the whole dependency managment in and we don't want that in client bom.